C++/Logger/conversion/Stringifiable.h

```diff
@@ -2,17 +2,20 @@
 #define _STRINGIFIABLE_H_
 
 #include "ConvertibleTo.h"
+#include "../metaprogramming/value_type_is.h"
 
 #include <string>
 
 namespace Toolbox::Log::Conversion
 {
 
-class Stringifiable : public ConvertibleTo<std::string>
+class Stringifiable
+  : public ConvertibleTo<std::string>
+  , value_type_is <std::string>
 {
 public:
   virtual const std::string toString (void) const = 0;
-  operator std::string (void) const override final;
```

Comment:
I see what you mean about there not being an implementation to begin with, but it will still have the above mentioned benefits and there really isn't a drawback.